tools/certs/Makefile

```diff
@@ -113,7 +118,7 @@ root-key.pem:
 %/ca-cert.pem: %/cluster-ca.csr k8s-root-key.pem k8s-root-cert.pem
 	@echo "generating $@"
 	@openssl x509 -req -days $(INTERMEDIATE_DAYS) \
-		-CA k8s-root-cert.pem -CAkey k8s-root-key.pem -set_serial $(INTERMEDIATE_SERIAL) \
+		-CA k8s-root-cert.pem -CAkey k8s-root-key.pem -CAcreateserial\
```

Comment:
A small comment on what that does ? I don't remember, I guess others will not know either. 
Is this the one where they store some files in the directory ?

Keep in mind multiple admins may run this command.

Reply:
This means openssl will generate serial number randomly  (and increased in future activations)